include/pros/misc.h

```diff
@@ -723,6 +723,26 @@ double battery_get_capacity(void);
  */
 int32_t usd_is_installed(void);
 
+/**
+ * Lists the files in a directory specified by the path
+ * Puts the list of file names (NOT DIRECTORIES) into the buffer seperated by newlines
+ * 
+ * \note use a path of "\" to list the files in the main directory
+ * 
+ * \return 1 on success or PROS_ERR on failure setting errno
```

Comment:
What is a failure?? What do the different errno codes mean?

I assume it can fail because either path doesn't exist, or the buffer was too small for the list, or maybe one of the pointers was null (maybe would just segfault there though...)

Any way to explain how to figure out why it failed? This would actually be very helpful to distinguish failure reasons, which matter here more than in most cases for the API.

Reply:
I will add more details to the errno section. I ran out of time before meetings and wanted to get this up tonight